src/thor/trace_route_action.cc

```diff
@@ -162,7 +162,7 @@ void thor_worker_t::route_match(Api& request) {
     thor::TripLegBuilder::Build(options, controller, *reader, mode_costing, path.begin(), path.end(),
                                 *options.mutable_locations()->begin(),
```

Comment:
this is all just adding algorithm names to the leg otherwise just some standardization of how we add a leg to a route